pdx-extract/tests/test_sanitize.py

```diff
@@ -0,0 +1,209 @@
+from app import sanitize
+from tests import data
+
+
+def test_get_courses():
+    subjects_json = data.subjects_json
+    courses = sanitize.get_courses(subjects_json)
+
+    assert len(courses) == 2
+    assert courses[0]["number"] == "CS 161"
+    assert courses[1]["number"] == "CS 457"
+    assert courses[0]["instructor"] == "David D Ely"
+    assert courses[1]["instructor"] == "Mark P Jones"
+
+
+def test_get_course_data_returns_sanitized_data():
+    # TODO: separate this into smaller tests
+    sanitized_course = sanitize.get_course_data(data.original_course)
```

Comment:
This seems fine not broken into smaller tests, but it looks like your test implementation relies heavily on the data it's accessing. Ideally, the test data would include expected results as well, so updating your data doesn't break the test.